### PR TITLE
new tutorial blog post

### PR DESCRIPTION
**Pull request by jukent**
I am creating a new blog post with tutorial dates that includes a new sign up sheet. This is needed so we can advertise in the Daily Bulletin and in Staff Notes Daily more effectively for the upcoming dates.

### CODE REVIEW

File: site/posts/python-tutorial-seminar-series-spring-2021.ipynb [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "_March 10, 2021_ - Numpy <br>\n",
+    "An introduction to essential Numpy functionality.\n",
+    "\n",
+    "_March 24, 2021_ - Matplotlib <br>\n",
+    "Learn basic plotting and data visualization techniques using the Matplotlib package.\n",
+    "\n",
+    "_April 14, 2021_ - Object Oriented Programming <br>\n",
+    "An introduction to Pyton \"objects\" such as classes.\n",
```
Comment (andersy005) [inline site/posts/python-tutorial-seminar-series-spring-2021.ipynb:22]:
```suggestion
    "An introduction to Python \"objects\" such as classes.\n",
```

Comment (jukent) [reply site/posts/python-tutorial-seminar-series-spring-2021.ipynb]: Thanks!

**Review by andersy005 — COMMENTED** [review]
@jukent, I presume the meeting times for these sessions are still 1 PM (Mountain Time). Would it be useful to add this info here?

**Review by andersy005 — APPROVED** [review]
Looks good to me 👍

**Comment by jukent** [comment]
I'm going to merge this in 1 hour unless there are any objections @NCAR/xdev-education because I do think that getting the announcement out is time sensitive.

**Pull request merged**